<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnSmtpSettingsWidget</class>
 <widget class="QWidget" name="QnSmtpSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="advancedCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Advanced settings...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="simplePage">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="simpleEmailLabel">
         <property name="text">
          <string>&amp;Email:</string>
         </property>
         <property name="buddy">
          <cstring>simpleEmailLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="simpleEmailLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="simplePasswordLabel">
         <property name="text">
          <string>&amp;Password:</string>
         </property>
         <property name="buddy">
          <cstring>simplePasswordLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="simplePasswordLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="detectErrorLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="horizontalFrame">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="serverLabel">
              <property name="text">
               <string>SMTP &amp;Server:</string>
              </property>
              <property name="buddy">
               <cstring>serverLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="serverLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="userLabel">
              <property name="text">
               <string>&amp;User:</string>
              </property>
              <property name="buddy">
               <cstring>userLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="userLineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="passwordLabel">
              <property name="text">
               <string>&amp;Password:</string>
              </property>
              <property name="buddy">
               <cstring>passwordLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="passwordLineEdit">
              <property name="echoMode">
               <enum>QLineEdit::PasswordEchoOnEdit</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="portLabel">
              <property name="text">
               <string>P&amp;ort</string>
              </property>
              <property name="buddy">
               <cstring>portComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="portComboBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="tlsRecommendedLabel">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;(recommended)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="unsecuredRadioButton">
           <property name="text">
            <string>Unsecure connection</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="tlsRadioButton">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Secure connection using TLS</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>advancedCheckBox</tabstop>
  <tabstop>simpleEmailLineEdit</tabstop>
  <tabstop>simplePasswordLineEdit</tabstop>
  <tabstop>serverLineEdit</tabstop>
  <tabstop>portComboBox</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>tlsRadioButton</tabstop>
  <tabstop>unsecuredRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
